README.md

```diff
@@ -28,7 +28,7 @@ You will find contribution documentation [here](https://github.com/Destronia/AdB
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+Thanks goes to these wonderful people (code):
```

Comment:
I'd remove the "(code)" section, ive checked the documents and i dont think thats how it works

Reply:
> I'd remove the "(code)" section, ive checked the documents and i dont think thats how it works

Oh ok
I dont think its important anyway, ill remove it.

I appreciate the information tho.

this'll do:
```suggestion
Thanks goes to these wonderful people:
```